third_party/repo.bzl

```diff
@@ -0,0 +1,160 @@
+# Copyright 2017 The TensorFlow Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities for defining TensorFlow Bazel dependencies."""
```

Comment:
These aren't TF deps anymore right, just flatbuffer? Maybe remove "tensorflow" and "tf" from this file and deps?

Reply:
I'm going to leave these as is for the moment because we are actually still using TF's mirrors. I suspect that will continue for the future (why have two sets of mirrors).